configure.ac

```
@@ -11,6 +11,23 @@ AC_DEFUN([AX_WITH_COMMENT], [
 	AC_ARG_WITH([ax_comment_$1],[$2],[],[])
 	])
 
+AC_SUBST(INSTALL_DOMSERVER,no)
+AC_ARG_ENABLE([domserver],AS_HELP_STRING([--enable-domserver],
+ [configure the domserver (default: no).]),
```

I'd suggest:
```suggestion
 [configure and build the domserver (default: no).]),
```
Same below for judgehost.